### PR TITLE
Admin Dashboard

### LINKED ISSUE #159: Admin Dashboard
Develop an Admin Dashboard that provides a high-level overview of key metrics, activities, and management controls to allow administrators to efficiently monitor and manage the system.

Key Tasks:

1. Design an intuitive and user-friendly layout for the admin dashboard. (Consider or improve the designed design in the Figma)
2. Integrate data from various parts of the system 
3. Use charts, graphs, and tables to visually represent important data
4. Ensure the dashboard layout is fully responsive and works smoothly across desktop, tablet, and mobile devices.

### PR DESCRIPTION
**Pull request by Mugisha146**
### PR Description
This PR  Develop Admin Dashboard to provides a high-level overview of key metrics, activities, and management controls to allow administrators to efficiently monitor and manage the system.

### Description of tasks that were expected to be completed
- Design an intuitive and user-friendly layout for the admin dashboard
- Integrate data from various parts of the system
- Use charts, graphs, and tables to visually represent important data
- Ensure the dashboard layout is fully responsive and works smoothly across desktop, tablet, and mobile devices.

### Locally 
- Clone repo
- Checkout branch `159-admin-dashboard`
- Then start the server `npm run dev` 
- Login as superAdmin or admin with credentials you have:

### Using deployed link(Preview):
Link : https://atlp-devpulse-fn-git-159-adm-692da8-devpulses-projects-62c56cf1.vercel.app/#/login
- Login as superAdmin or admin with credentials : 
            Email : admin@example.com
            Password : password123

### Track PR (issue number & link)
Issue #159

### CODE REVIEW

**Review by MnorbertVii — CHANGES_REQUESTED** [review]
hi @Mugisha146 I noticed that the time displayed in Overall performance statistics section is currently static and does not update in real-time. This could lead to confusion for users as they may think the information is up to date when it’s not.

**Comment by Mugisha146** [comment]
> hi @Mugisha146 I noticed that the time displayed in Overall performance statistics section is currently static and does not update in real-time. This could lead to confusion for users as they may think the information is up to date when it’s not.

Now Solved, Thanks

**Review by mugishaj092 — CHANGES_REQUESTED** [review]
solve codeclimate issue and remember to add credentials  work on norberts feedback

**Comment by MnorbertVii** [comment]
solve code climate issues

**Review by MnorbertVii — APPROVED** [review]

**Review by mutsinziisaac — APPROVED** [review]

**Review by mugishaj092 — APPROVED** [review]

**Review by Ishimwe7 — APPROVED** [review]
Well done @Mugisha146

**Comment by ncutixavier** [comment]
This PR has conflicts

**Comment by Mugisha146** [comment]
> This PR has conflicts

Solved

**Review by ncutixavier — APPROVED** [review]

**Comment by ncutixavier** [comment]
This PR has many commits

**Comment by Mugisha146** [comment]
> This PR has many commits

Solved, Now it has 1 commit

File: src/redux/actions/users.ts [hunk 1]
```diff
@@ -88,4 +88,40 @@ export const assignMemberRoles= async (userId, roleId)  => {
...
+    );
+
+    dispatch({
+      type: "FETCH_COORDINATORS",
+      data: coordinators,
+    });
+
+    return coordinators.length;
+  } catch (err) {
+    console.error("Error fetching coordinators:", err);
```
Comment (ncutixavier) [inline src/redux/actions/users.ts]: Remove all logs

**Review by ncutixavier — COMMENTED** [review]
Please remove all unnecessary logs

File: src/redux/actions/users.ts (hunk 1 above)
Comment (Mugisha146) [reply src/redux/actions/users.ts]: Done

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**